std/assembly/util/error.ts

```diff
@@ -9,6 +9,10 @@ export const E_INDEXOUTOFRANGE: string = "Index out of range";
 @lazy @inline
 export const E_INVALIDLENGTH: string = "Invalid length";
 
+// @ts-ignore: decorator
+@lazy @inline
+export const E_ILLEGALGENTYPE: string = "Illegal generic type";
```

Comment:
add `generic` due to `TypeError` usually already relate to Illegal type. So I little bit specialize this